src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilsTest.java

```diff
@@ -114,6 +118,31 @@ public void testBadRegex2() {
     }
 
     @Test
+    public void testCreationOfFakeCommentBlock() throws Exception {
+        final Method createFakeBlockComment =
```

Comment:
As method became public there is no need in reflection.

Reply:
fixed